indra/newview/lloutfitgallery.cpp

```diff
@@ -1251,7 +1273,15 @@ void LLOutfitGalleryGearMenu::onUpdateItemsVisibility()
 {
     if (!mMenu) return;
     bool have_selection = getSelectedOutfitID().notNull();
+    mMenu->setItemVisible("expand", false);
+    mMenu->setItemVisible("collapse", false);
     mMenu->setItemVisible("thumbnail", have_selection);
+    mMenu->setItemVisible("inventory_settings", false);
```

Comment:
Where did all this come from?

Reply:
b9633c17e373bfe55b29228996e8473eb041466d

Reply:
Then you missed 98f7d73d46fdc045759023eda6409e8c791f5cb2

Reply:
Thank you, checking..